### PR TITLE
[MODULAR] Readds three loadout items, Flowers are correctly Donator-only again

### PR DESCRIPTION
**Pull request by ArdentSpark**
## About The Pull Request

Adds the black and red victorian dress, adds the poly sweater for non-donators, removes duplicated flower entries in the loadout and moves all of them to donator only.

## How This Contributes To The Skyrat Roleplay Experience

Missing loadout items bad. Fashion good.
Duplicate flowers in code bad. Caused duplicate error messages for admins. Errors bad.
Flowers were available to non-donators. Donators feeling validated for their contributions good. Free stuff bad.

## Changelog

:cl:
fix: Readding missing loadout items, and removing items from non-donator access as it was previously 
fix: Adds Red Victorian Dress, Black Victorian Dress and Polychromic Sweater.
fix: Migrates Poppy, Lily, Geranium, Rose, Sunflower, Harebell and Rainbow Bunch to donator only once more, reducing the admin spam.
/:cl:

### CODE REVIEW

**Comment by ArdentSpark** [comment]
The Polychromatic Sweater has been reported to have been accessible to Non-donators before the character creator remodel, yet the item is stored within a donator clothing file. 

Can I get your opinion on moving it to non-donators before merging?

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me. I'm not 100% sure why the polychromic sweater is donator-only, but if that's how it was before, I guess I don't have anything against it.

**Comment by ArdentSpark** [comment]
Deleted a bunch of random duplicate empty lines in 2ba6c05 as well. But otherwise I'm done tinkering with this PR now, I'll start a new one for further work going forward if anything comes up

**Comment by ArdentSpark** [comment]
haha funny wallet duplicate failed CI time to revert

File: modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm [hunk 1]
```diff
@@ -390,6 +391,14 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	name = "Villain Suit"
 	item_path = /obj/item/clothing/under/costume/villain
 
+/datum/loadout_item/under/miscellaneous/costume/vic_dress
+	name = "Black victorian dress"
+	item_path = /obj/item/clothing/under/costume/vic_dress
+
+/datum/loadout_item/under/miscellaneous/costume/vic_dress_red
```
Comment (thestubborn) [inline modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]: these should be under the formal stuff, tbh

Comment (ArdentSpark) [reply modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]: Probably not wrong, I just allocated it based on it's item path

Comment (thestubborn) [reply modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]: idk
its more a formal getup to me but ydy

Comment (ArdentSpark) [reply modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]: I'll definitely take a look and sanity check it's location before I mark as resolved

**Comment by softcerv** [comment]
One potential issue that I could see with this is that poppies have 20% libital which technically means that they can be used as healing items. Granted, I haven't really seen that ever used in practice.

**Comment by ArdentSpark** [comment]
I agree softcerv, and in fact I raised this concern when Ashwalkers were able to spawn with loadouts including poppies. And even in that instance, ashwalkers have vulgaris which has libital and aiuri. Poppies are also available in the public garden on all (IIRC) maps, but don't quote me. 
However regarding that, they were always a donator item, they just broke in the loadout transfer because someone added them as donator as well as non donator incorrectly. If there are any concerns about donator status items then I'll work with whatever's decided but I've been bouncing off of Golden here

File: modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm (hunk 1 above)
Comment (ArdentSpark) [reply modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm]: resolved in 158281a

**Comment by softcerv** [comment]
> 
> 
> I agree softcerv, and in fact I raised this concern when Ashwalkers were able to spawn with loadouts including poppies. And even in that instance, ashwalkers have vulgaris which has libital and aiuri. Poppies are also available in the public garden on all (IIRC) maps, but don't quote me. However regarding that, they were always a donator item, they just broke in the loadout transfer because someone added them as donator as well as non donator incorrectly. If there are any concerns about donator status items then I'll work with whatever's decided but I've been bouncing off of Golden here

That's fair enough. I figure that I'd bring it up now while there is a PR made involving the flower load-out items, even if my concerns aren't in the scope of this PR. I mostly mentioned it because, restoring it back to a donator item is technically giving paying players an advantage to some degree. (although, that advantage is very minute and most people don't use it.)

**Comment by tf-4** [comment]
Could make a reagent-free subtype and call it "artificial poppy" or whatever.

**Comment by GoldenAlpharex** [comment]
> Could make a reagent-free subtype and call it "artificial poppy" or whatever.

Or just call them poppies and don't care, players don't have to know.

**Comment by softcerv** [comment]
> 
> 
> > Could make a reagent-free subtype and call it "artificial poppy" or whatever.
> 
> Or just call them poppies and don't care, players don't have to know.

That'd be a nice solution to be honest, maybe I should look into coding that later.

**Review by GoldenAlpharex — APPROVED** [review]
I'm good with this, we can fix the flower later, especially since I don't think I've ever actually seen anyone take it for that. Regardless, it's a good point, but the bugfix is more important than fixing a potential player issue in the future, for now.

**Comment by GoldenAlpharex** [comment]
Just make an issue about it so we don't forget it @softcerv :)

**Comment by tf-4** [comment]
> 
> 
> > Could make a reagent-free subtype and call it "artificial poppy" or whatever.
> 
> Or just call them poppies and don't care, players don't have to know.

well it would avoid confusion with people grinding poppies and expecting reagents

**Pull request merged**